docs/install/create-project-cookieplone.md

```diff
@@ -189,8 +189,8 @@ before. Is it okay to delete and re-download it? [y/n] (y):
   [5/17] Author (Plone Foundation): 
   [6/17] Author E-mail (collective@plone.org): 
   [7/17] Should we use prerelease versions? (No): 
-  [8/17] Plone Version (6.1.0): 
-  [9/17] Volto Version (18.8.1): 
+  [8/17] Plone Version (6.1.0): (press enter if it is your first project)
```

Comment:
This is literal console output from running the command. We can't add something here that doesn't actually happen. Cookieplone would need to be updated with this altered prompt first.

Alternatively you can move instructions into the narrative documentation with the other admonitions above.